sites/packworld/server/templates/index.marko

```diff
@@ -29,11 +30,74 @@ $ const { id, alias, name, pageNode } = data;
 
     <marko-web-query|{ nodes }|
       name="website-optioned-content"
-      params={ sectionId: id, optionId: 30004, limit: 5, requiresImage: true, queryFragment }
+      params={ sectionId: id, optionName: "Pinned", limit: 5, requiresImage: true, queryFragment }
     >
       <website-content-hero-flow nodes=nodes />
     </marko-web-query>
 
+    <div class="row">
+      <div class="col-lg-8 mb-block">
+        <marko-web-query|{ nodes }|
+          name="website-optioned-content"
+          params={ sectionAlias: "home", optionName: "Pinned", limit: 2, skip: 5, requiresImage: true, queryFragment }
+        >
+          <default-theme-card-deck-flow cols=2 nodes=nodes>
+            <@slot|{ node, index }|>
+              <website-content-card-node
+                image-width=340
+                node=node
+              />
+            </@slot>
+          </default-theme-card-deck-flow>
+        </marko-web-query>
+
+        <div class="row">
+          <div class="col-lg-6 mb-block">
+            <marko-web-query|{ nodes }|
+              name="website-optioned-content"
+              params={ sectionAlias: "home", optionName: "Pinned", limit: 1, skip: 7, requiresImage: true, queryFragment }
```

Comment:
Add `contentType=['Podcast']`, split params to new lines, remove skip from this query.